### PR TITLE
GraphQL and AsyncAPI changes in kickstart

### LINKED ISSUE #537: GraphQL and AsyncAPI changes for kickstart
## Is your feature request related to a problem? Please describe
GraphQL Doc related changes in kickstart

Related to https://github.com/apigee/apigee-api-catalog-drupal/issues/140

### PR DESCRIPTION
**Pull request by phdhiren**

### CODE REVIEW

File: apigee_devportal_kickstart.install [hunk 1]
```diff
@@ -607,3 +607,48 @@ function apigee_devportal_kickstart_update_8011() {
...
+    }
+
+    $filters['type_1']['table'] = 'node_field_data';
+    $filters['type_1']['field'] = 'type';
+    $filters['type_1']['entity_type'] = 'node';
+    $filters['type_1']['entity_field'] = 'type';
+    $filters['type_1']['plugin_id'] = 'bundle';
+    $filters['type_1']['operator'] = 'in';
+    $filters['type_1']['value']['apidoc'] = 'apidoc';
+    if (\Drupal::moduleHandler()->moduleExists('apigee_graphql_doc')) {
```
Comment (giteshk) [inline apigee_devportal_kickstart.install:638]: Should this be if module is enabled?

**Review by giteshk — COMMENTED** [review]
Want to make sure GraphQL and Async are not automatically enabled during the profile install.

File: apigee_devportal_kickstart.install (hunk 1 above)
Comment (phdhiren) [reply apigee_devportal_kickstart.install:638]: Yes it is for enabled modules

**Review by divya-intelli — APPROVED** [review]
LGTM

**Pull request merged**